src/pyhf/infer/__init__.py

```diff
@@ -17,6 +24,7 @@ def hypotest(
         pdf (~pyhf.pdf.Model): The HistFactory statistical model
         init_pars (Array or Tensor): The initial parameter values to be used for minimization
         par_bounds (Array or Tensor): The parameter value bounds to be used for minimization
+        fixed_vals (list of tuples): Parameters to be held constant and their value
```

Comment:
need to be careful, since this will be confusing given the existing `poi_test` argument.

Reply:
It can be confusing because `fixed_poi_fit` is not strictly necessary anymore. Anyway in `fixed_poi_fit` the mu is fixed via the `fixed_vals` and if additional parameters are set to constant they're added to the list, see https://github.com/cgottard/pyhf/blob/fixed_vals/src/pyhf/infer/mle.py#L56
So there should be no problem nor ambiguity.